todo_app/app.py

```diff
@@ -8,10 +8,21 @@
 from todo_app.models.view_model import ViewModel
 from flask_login import LoginManager, login_required
 
+from loggly.handlers import HTTPSHandler 
+from logging import Formatter
+
+
 def create_app(database_name="Main"):
     app = Flask(__name__)
-    app.config["SECRET_KEY"]=os.environ.get("SECRET_KEY")
+    app.config["SECRET_KEY"] = os.environ.get("SECRET_KEY")
     app.config["USER_UNAUTHORIZED_ENDPOINT"] = "/index"
+    app.logger.setLevel(os.environ.get("LOG_LEVEL", default="DEBUG"))
```

Comment:
A default of DEBUG makes sense in your `.env.template` as that's the local environment default, but here INFO might make more sense to ensure that production stays at a sensible level even if the relevant environment variable was missed (e.g. to match your terraform default)